x-pack/plugin/ent-search/qa/full-cluster-restart/javaRestTest/java/org/elasticsearch/entsearch/FullClusterRestartIT.java

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+package org.elasticsearch.entsearch;
```

Comment:
It seems to me that we should use `org.elasticsearch.xpack.application` instead of `org.elasticsearch.entsearch`.
Also I would prefer to organize those tests by application (`analytics` here). 

```suggestion
package org.elasticsearch.xpack.application.analytics;
```

Reply:
I kept it as `entsearch` assuming that we'll have additional cases in the module outside of BA in the future that we want to test a full restart with.

Reply:
There is two different points here.

First `org.elasticsearch.entsearch` has been abandoned since a while and your package name should at least be `org.elasticsearch.xpack.application`.

Second, I would like to keep the code (including tests) of each application separated since it ease refactoring. If you want to add a test for another application (let's say for search application), I would recommend that is goes into it own class which would be in the right package.

Reply:
Thanks for the explanation. I have refactored the package name, but as discussed elsewhere I'd like to keep the pattern now of using `FullClusterRestartIT`. We can refactor this at a later date if it becomes burdensome.